xfade.ttl.in

```diff
@@ -16,8 +17,9 @@
 	a lv2:Plugin, lv2:MixerPlugin, doap:Project;
 	doap:license <http://usefulinc.com/doap/licenses/gpl> ;
 	doap:maintainer <http://gareus.org/rgareus#me> ;
-	doap:name "Stereo DJ X-Fade";
+	doap:name "Stereo X-Fade";
 	@VERSION@
+	mod:brand "x42" ;
```

Comment:
This should become a `@MODBRAND@`  and `@MODLABEL@` which are replaced during build.